README.md

```diff
@@ -50,6 +50,7 @@ python app.py
 
 5. Go to ` http://127.0.0.1:5000/` and enjoy the application.
 
+
```

Comment:
Is there any particular need of this extra line here..?

Reply:
Otherwise LGTM :rocket:

Reply:
Actually I also noticed the same. I'll remove that extra line and will merge then.